es62x/build.gradle

```diff
@@ -51,11 +51,9 @@ compileScala {
 
 dependencies {
```

Comment:
what about es63x?

Reply:
I missed it, because it because only here log4j is in version '2.11.1'. Shouldn't it be propagated to other modules?